doc/api/python.rst.py

```diff
@@ -36,9 +36,15 @@
         # Do not show in docs when no doc string is present
         pass
 
+# Document eos.tasks automatically
+# Excluded tasks must be handled explicitly
+excluded_tasks = ['find_mode', 'run']
```

Comment:
Any reason why you exclude `find_mode` as well?

Reply:
Because it is treated explicitly in the documentation.
Why this is the case, I am not sure! There is not special commentary, so it might just be a remnant of history.

Reply:
As discussed in the group meeting: Can you merge the two `tasks` section since `Mode Finding` does not contain any additional information?